docs/user-guide/side-bar.md

```diff
@@ -1,94 +1,91 @@
+Trought the *Show my position* <img src="../img/button/geolocation.jpg" class="ms-docbutton"/> the user can center the map on his position. Therefore the button turns green. 
 
 <img src="../img/side-bar/position.jpg" class="ms-docimage"/>
 
+The position is still active even when the user interacts with the map; with a single click on the button it is possible re-center the map on his position. 
+To desable the position the button needs to be duble clicked. 
+
 Zooming tools
 -------------
 
-[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) provides several tools to zoom in and out, and to switch the map extent.
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) provides several tools allows the user to:
 
-* You can increase the map zoom by using the zoom in icon <img src="../img/button/zoom-in.jpg" class="ms-docbutton"/>.
+* **Increase** <img src="../img/button/zoom-in.jpg" class="ms-docbutton"/> the map zoom
 
-* You can decrease the map zoom by using the zoom out icon <img src="../img/button/zoom-out.jpg" class="ms-docbutton"/>.
+* **Decrease** <img src="../img/button/zoom-out.jpg" class="ms-docbutton"/> the map zoom
 
-* You can switch to full screen view <img src="../img/button/full-screen.jpg" class="ms-docbutton"/>.
+* **Switch to full screen**  <img src="../img/button/full-screen.jpg" class="ms-docbutton"/> view
 
-* You can navigate through the extents used in a session by going back <img src="../img/button/back-extent.jpg" class="ms-docbutton"/> and forward <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> or to zoom to the maximum extent <img src="../img/button/max-extent.jpg" class="ms-docbutton"/>.
-
-3D Navigation
--------------
+* **Go back** <img src="../img/button/back-extent.jpg" class="ms-docbutton"/> to the last change 
 
-Based on CesiumJS, a 3D Navigator is implemented in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) that allows you to display the map contents in 3D mode, particularly the layers will be overlaid on the globe.
+* **Go forward** <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> to the last change 
 
-* **Open** a new map then **Add** a layer.
+* **Zoom to the maximum extent** <img src="../img/button/max-extent.jpg" class="ms-docbutton"/> the map
 
-* **Click** on the 3D icon <img src="../img/button/3D-icon.jpg" class="ms-docbutton"/> in the *Sidebar*.
+3D Navigation
+-------------
 
-The scene will switch from 2D to 3D display mode and you will be guided by a brief tutorial that explain you how to interact with the map.
+Based on CesiumJS, a **3D Navigator** <img src="../img/button/3D-icon.jpg" class="ms-docbutton"/> is implemented in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) that allows the user to display the map contents in 3D mode. In particularly the layers will be overlaid on the globe and it is possible to orbit around the globe trought the compass place in the upper right corner of the map.
```

Comment:
The 3D navigation in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is based on CesiumJS. If the 3D button <img src="../img/button/3D-icon.jpg" class="ms-docbutton"/> in the sidebar is clicked, the map switch in 3D mode so map contents are displayed on a 3D globe and it is possible to orbit around it **through** the compass place in the upper right corner of the map.